post-scan-actions/azure-python-promote-or-quarantine/template.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
+    "contentVersion": "1.0.1.0",
```

Comment:
This is template version, so I think in this case it doesn't need to be updated.